controller/engine_controller.go

```diff
@@ -1126,12 +1127,12 @@ func syncWithRestoreStatus(log logrus.FieldLogger, engine *longhorn.Engine, rsMa
 	}
 	if isConsensual {
 		if engine.Status.LastRestoredBackup != lastRestored {
-			log.Infof("Update last restored backup from %v to %v", engine.Status.LastRestoredBackup, lastRestored)
+			log.Infof("Updated last restored backup from %v to %v", engine.Status.LastRestoredBackup, lastRestored)
```

Comment:
`Updating last ...` because this log is before the following update.

Reply:
Modified.